app/views/posts/show.html.erb

```diff
@@ -1,2 +1,21 @@
 
 <h1>Here show post for a given use</h1>
+
+      <div class="row mt-3">
+         <div class="col-12 border ">
+              <div class ="d-flex justify-content-between">
+               <h1><%= @post.title %> by <%= @post.title %></h1>
+            <div class ="d-flex flex-row-reverse">
+                <p class="text-end p-2">likes: <%= @post.likes_counter %></p>
+                <p class="text-end p-2">comments: <%= @post.comments_counter %></p>
+            </div>
+          </div>
+            <p><%= @post.text %></p>
+      </div>
+      <% @commit.each do |commit| %>
+      <div class="row mt-3">
+          <div class="col-12">
+              <p><%= commit.author_id %>: <%= commit.text %></p>
+          </div>
+      </div>
+      <% end %>
```

Comment:
- [ ] The posts show page doesn't have margin and border that can be seen in the wireframe. You should update the styling of this page to make it more look like the wireframe. Please compare the below screenshot with the wireframe.


| Screenshot | Wireframe |
| --------------- | -------------- |
| ![Screenshot 2022-01-26 at 01-20-00 Blog](https://user-images.githubusercontent.com/1642014/151072213-7a082b0c-612a-4fef-9270-403d3858e5d2.png)  | ![Wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAABQJEXGVTGMQQALKTWUTMM4OYPZXADQ) |